### PR TITLE
optimize updateServiceUrl method

### PR DESCRIPTION
**Pull request by lordcheng10**
### Motivation
optimize updateServiceUrl method

### Documentation

Check the box below or label this PR directly (if you have committer privilege).

Need to update docs? 

- [ ] `doc-required` 
  
  (If you need help on updating docs, create a doc issue)
  
- [x] `no-need-doc` 
  
  (Please explain why)
  
- [ ] `doc` 
  
  (If this PR contains doc changes)

### CODE REVIEW

**Comment by lordcheng10** [comment]
@hangc0276 PTAL,thanks!

File: pulsar-client/src/main/java/org/apache/pulsar/client/impl/PulsarClientImpl.java [hunk 1]
```diff
@@ -860,6 +860,10 @@ public boolean isClosed() {
 
     @Override
     public synchronized void updateServiceUrl(String serviceUrl) throws PulsarClientException {
+        if (serviceUrl != null && serviceUrl.equals(conf.getServiceUrl())) {
+            log.info("Ignore service URL to {}，because it is the same as the current service URL!", serviceUrl);
```
Comment (Technoboy-) [inline pulsar-client/src/main/java/org/apache/pulsar/client/impl/PulsarClientImpl.java:864]: `Ignore service URL` -> 'Ignore update service URL' ?

Comment (lordcheng10) [reply pulsar-client/src/main/java/org/apache/pulsar/client/impl/PulsarClientImpl.java:864]: image
Aha, I made a mistake, here is the judgment of whether the url is equal.  I will close this PR

**Pull request closed without merging**